fabio.properties

```diff
@@ -743,6 +743,17 @@
 # registry.consul.checksRequired = one
 
 
+# glob.matching.enabled Enables glob matching on route lookups
+# If glob matching is enabled there is a performance decrease
+# for every route lookup.  At a large number of services (> 500) this
+# can have a significant impact on performance. If glob matching is disabled
+# Fabio performs a static string compare for route lookups.
+#
+# The default is
+#
+# glob.matching.enabled = true
```

Comment:
I think they are similar but we do not use URI matching.  Just Host Matching.  We are looking for this feature to avoid using Globs at all and use straight string matching on the host.

Reply:
Makes sense